src/Sarif.Driver/Sdk/SkimmerBase.cs

```diff
@@ -82,8 +82,6 @@ internal static string FirstSentence(string fullDescription)
 
         public IDictionary<string, string> Options { get; }
```

Comment:
Sure, that sounds good.

---
In reply to: [62771555](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62771555) [](ancestors = 62771555)